content/blog/blog-2020-07-30-introducing-operators-index.md

```diff
@@ -0,0 +1,85 @@
+---
+date: 2020-07-30
+---
+
+# Introducing the KUDO Operators Index
+
+The default repository when installing operators with KUDO is the community repository. Adding new operators to this repository has been greatly simplified with a new index of operators and a new community repository URL that is used starting with KUDO 0.16.
+
+<!-- more -->
+
+Adding new operators to the community repository was traditionally done by adding the operator packages as a folder in the [operators](https://github.com/kudobuilder/operators) project. While this worked well for smaller operators, it created challenges for large operators, for example, [KUDO Cassandra](https://github.com/mesosphere/kudo-cassandra-operator). Larger operators are developed as separate projects in their own git repository. For new releases, the operator package would have to be copied over to the _operators_ project. In case of bugs in the operator, is the source of truth now the origin project or the folder in the _operators_ project?
```

Comment:
Maybe a sentence about *why* bigger operators have their own repository?

- Issue tracking
- Documentation
- Extended Tests
- Non-Operator things: Docker images, build and release processes